pkg/promtail/targets/filetarget.go

```diff
@@ -298,15 +286,18 @@ func toStopTailing(nt []string, et map[string]*tailer) []string {
 	return ta
 }
 
-func (t *FileTarget) updateTotalBytesMetric(ms []string) {
+func (t *FileTarget) reportSizeAndRemoveMissing(ms []string) []string {
+	mso := ms[:0]
 	for _, m := range ms {
 		fi, err := os.Stat(m)
 		if err != nil {
-			level.Error(t.logger).Log("msg", "failed to stat matched file, cannot report size", m, "error", err)
+			//If we can't stat the file, skip it
```

Comment:
I'm curious why do you removed the log ? is the reason for that error always the same and not important ?

Reply:
because locally my kubernetes had broken symlinks in the `/var/pods/...` and it just ends up filling the logs with error messages every 10 seconds when the sync function runs...

It's a good question though, not sure why this happened and if we ignore the error what the consequence of that is... maybe we should leave it as level.Warn? and get a scope of if the problem even exists in our k8 clusters?

Reply:
I would leave it with a level just below the default log level. So if someone want to investigate he might stumble on it.